src/main/java/uk/gov/hmcts/reform/pip/data/management/service/LocationService.java

```diff
@@ -225,12 +225,14 @@ public byte[] downloadLocations() throws IOException {
                         location.getName(),
                         location.getRegion(),
                         location.getJurisdiction(),
+                        location.getJurisdictionType(),
```

Comment:
Can the LocationServiceTest unit test be updated to check for this

Reply:
Updated